h/browser/chrome/lib/uri-info.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * A wrapper for the server's uriinfo endpoint.
+ *
+ * Exports a function that makes an HTTP request to the server's uriinfo
+ * endpoint and returns a Promise that resolves to the response. Usage:
+ *
+ *     var uriInfo = require('uri-info');
```

Comment:
> I've corrected some other places where uri-info.js was imported as
uriinfo (lowercase) though

That's what I meant :)

On 15 October 2015 at 13:05, Sean Hammond <notifications@github.com> wrote:

> In h/browser/chrome/lib/uri-info.js
> <https://github.com/hypothesis/h/pull/2610#discussion_r42115407>:
>
> > @@ -0,0 +1,120 @@
> > +/**
> > + * A wrapper for the server's uriinfo endpoint.
> > + *
> > + * Exports a function that makes an HTTP request to the server's uriinfo
> > + * endpoint and returns a Promise that resolves to the response. Usage:
> > + *
> > + *     var uriInfo = require('uri-info');
>
> I'm not sure what you want me to change here? This is consistent with
> other modules, eg. sidebar-injector.js is imported as SidebarInjector.
>
> I've corrected some other places where uri-info.js was imported as uriinfo
> (lowercase) though
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2610/files#r42115407>.
>